DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Extensions/SecurityContext/SecurityContextUserExtension.cs

```diff
@@ -8,10 +8,14 @@ namespace GitHubExtension.Security.WebApi.Extensions.SecurityContext
 {
     public static class SecurityContextUserExtension
     {
-        public static IEnumerable<User> GetUsersByName(this ISecurityContextQuery securityContextQuery, string userName)
+        public static IEnumerable<User> GetUsersByNameExceptCurrent(
```

Comment:
I'm sure :)
But I will add unit tests to prevent braking changes in the future.